src/huggingface_hub/utils/cache_explorer.py

```diff
@@ -239,8 +253,8 @@ def _format_size(num: int, suffix: str = "B") -> str:
 
     Taken from https://stackoverflow.com/a/1094933
     """
-    for unit in ["", "Ki", "Mi", "Gi", "Ti", "Pi", "Ei", "Zi"]:
-        if abs(num) < 1024.0:
+    for unit in ["", "K", "M", "G", "T", "P", "E", "Z"]:
+        if abs(num) < 1000.0:
             return f"{num:3.1f}{unit}{suffix}"
-        num /= 1024.0
+        num /= 1000.0
     return f"{num:.1f}Yi{suffix}"
```

Comment:
i think you need to remove the `i` here?

Reply:
Indeed, thanks for the catch 
Would have displayed a `YiB` in case someone has a file of 10^24 bytes 😁

Reply:
haha yeah that would be one large ML model